go/d/try.go

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, version 2.0:
 // http://www.apache.org/licenses/LICENSE-2.0
 
+// Package d implements several error and assertion functions used throughout Noms.
```

Comment:
`d` is shorter, essentially :-) Also, those of us from the Chromium world were used to using a macro called DCHECK() for stuff similar to this.